src/components/app/DocumentList.js

```diff
@@ -186,7 +186,9 @@ class DocumentList extends Component {
             this.setState({
                 cachedSelection: null
             }, () => {
-                dispatch(setListIncludedView());
+                if (includedView.windowType === 'pickingSlot') {
```

Comment:
I told that this is workaround. But doing this right requires significant rework of DocumentList and potentially has wide impact, so we need to do this carefully.

Distinct task for that would be good.